sdk/core/azure-core/azure/core/credentials.py

```diff
@@ -50,24 +50,24 @@ def __init__(self, key):
     @property
     def key(self):
         # type () -> str
-        """The value of the configured API key.
+        """The value of the configured key.
 
         :rtype: str
         """
         return self._key
 
     def update(self, key):
         # type: (str) -> None
-        """Update the API key.
+        """Update the key.
 
-        This can be used when you've regenerated your service API key and want
+        This can be used when you've regenerated your service key and want
         to update long-lived clients.
 
-        :param str key: The API key to your Azure account.
+        :param str key: The key to your Azure account.
```

Comment:
Let's update this description. Azure account makes me look for a key associated with all services. What are other languages using here?

Reply:
.NET uses "Key used to authenticate to an Azure service.", so I'll add "service" to the docstring to make it more specific